tests/warehouses/spark_suite.py

```diff
@@ -0,0 +1,9 @@
+from tests.common.sql_test_case import TARGET_SPARK
+from tests.common.sql_test_suite import SqlTestSuite
+
+
+class HiveSuite(SqlTestSuite):
+
+    def setUp(self) -> None:
+        self.target = TARGET_SPARK
```

Comment:
Maybe could be a class attribute. I copy pasted this from another suite.

Reply:
Good point, Will add to my refactor list.